app/src/main/java/it/ministerodellasalute/immuni/ui/countriesofinterest/CountriesFragment.kt

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (C) 2020 Presidenza del Consiglio dei Ministri.
+ * Please refer to the AUTHORS file for more information.
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package it.ministerodellasalute.immuni.ui.countriesofinterest
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.google.android.material.appbar.AppBarLayout
+import it.ministerodellasalute.immuni.R
+import it.ministerodellasalute.immuni.extensions.view.setSafeOnClickListener
+import it.ministerodellasalute.immuni.logic.exposure.CountriesOfInterestManager
+import it.ministerodellasalute.immuni.logic.exposure.models.CountryOfInterest
+import it.ministerodellasalute.immuni.ui.dialog.ConfirmationDialogListener
+import it.ministerodellasalute.immuni.ui.dialog.openConfirmationDialog
+import kotlinx.android.synthetic.main.countries_of_interest.*
+import org.koin.android.ext.android.inject
+import java.util.*
+import kotlin.math.abs
+
+class CountriesFragment :
+    Fragment(), CountriesClickListener, ConfirmationDialogListener {
+
+    private val countriesManager: CountriesOfInterestManager by inject()
```

Comment:
This breaks the codebase architecture conventions: Fragments should only communicate with ViewModels.
Minor, but a quick fix should inject the manager into the view model.